### DISCUSSION TITLE
Lack of Material Assigning Option

### DISCUSSION THREAD

**Discussion by Hosseinan9675** [Anura3D/Anura3D_OpenSource#259] [Q&A]
Hi, 
I am trying to develop a Slope Stability. I could Compile the 2024 Code on the Visual Studio 2022. I opened the Anura3D with the Gid17.01 version. It seems that the code I compiled was not complete! As can be seen in below I can't see the Option of assigning material! Would you please guide me to sort this out? 
![Screenshot 2024-09-18 210246](https://github.com/user-attachments/assets/bc54d09c-e00b-4b83-b2b7-5de4c6c12d8e)

**Comment by mozadst** [comment]
I have the same problem, have you figured this out yet?

**Comment by mikael-onsilva** [comment]
To assign a material, you must right-click on "2D Single-point", inside "Material Point Specification", then click on "Apply to entities", so you select the material and the region you want to apply it. Just follow the tutorial manual that you should be fine.
